datalad/core/distributed/clone.py

```diff
@@ -630,12 +633,14 @@ def postclonecfg_ria(ds, props):
         scheme = props['giturl'].split(':', 1)[0]
         if scheme in ['http', 'https']:
             try:
-                response = requests.get("{}{}config".format(
-                    props['giturl'],
-                    '/' if not props['giturl'].endswith('/') else '')
-                )
-                config_content = response.text
-            except requests.RequestException as e:
+                with TemporaryDirectory() as _dest:
```

Comment:
@mih: Looking at the failure in CrippledFS, using `TemporaryDirectory` might be the issue. We have dedicated helpers for temp locations that deal with several env vars/configs to figure out where exactly to put it.

Reply:
Update: Just like the interactive/authentication issue below, the CrippledFS failure actually seem to depend on moon phases and the color of my socks. *grmpf*
However, we ran into inconsistent temp location issues before, so I still think it's better to get that directory from the helpers.

Reply:
you can just fetch content via our downloaders (used by `download_url`) directly without saving into a file. E.g.
```
$> python -c 'from datalad.downloaders.providers import Providers; url="http://datasets.datalad.org/.git/config"; print(Providers.from_config_files().fetch(url))' | head
[INFO] Fetching 'http://datasets.datalad.org/.git/config' 
[core]
	repositoryformatversion = 0
	filemode = true
	bare = false
	logallrefupdates = true
	sharedrepository = 2
[receive]
...
```
so there is `.fetch` to complement `.download`, and thus you do not need any temp files AFAIK

Reply:
This is implemented now